koku/api/provider/models.py

```diff
@@ -178,10 +178,10 @@ class Meta:
     source_type = models.CharField(max_length=50, null=False)
 
     # Provider authentication (AWS roleARN, OCP Sources UID, etc.)
-    authentication = models.CharField(max_length=128, null=False)
+    authentication = JSONField(null=False, default=dict)
```

Comment:
How are we going to handle migrating existing rows? The migration doesn't appear to cover it, unless there's some sort of default.

Reply:
Since we don't have any data in CI or production I didn't bother making the migration change.

Reply:
I'm okay with that.